backend/src/models/connection.ts

```diff
@@ -2,6 +2,9 @@ import { createConnection, Connection } from 'typeorm';
 import {
   // Models for the Crossfeed database
   ApiKey,
+  SavedSearch,
```

Comment:
SavedSearch is a duplicate import. It is also being imported at line 19.

Reply:
I went ahead and resolved the duplicate issue myself and rebuilt. The change can be viewed on commit: c3af8b7375a248e9be731b1a20f00e01d9e43b21